scripts/templates/variables.tf.j2

```diff
@@ -64,7 +64,13 @@ variable "{{ id }}_disabled_{{ severity }}" {
   type        = bool
   default     = null
 }
-
+{% if rule.lasting is string %}
+variable "{{ id }}_lasting_{{ severity }}" {
```

Comment:
```suggestion
variable "{{ id }}_lasting_duration_{{ severity }}" {
```

to keep same naming than existing: https://github.com/claranet/terraform-signalfx-detectors/blob/master/modules/integration_aws-elb/variables.tf#L233

Reply:
Fixed in last commit.